src/odemis/gui/cont/menu.py

```diff
@@ -73,6 +73,17 @@ def __init__(self, main_data, main_frame):
                         main_frame.menu_item_reset_finealign.GetId(),
                         self._on_reset_align)
 
+        # Assign 'Reset overview' functionality (if the tab exists)
+        try:
```

Comment:
Instead of a try/except, just use `if main_data.role == "secom"`. (That will also avoid showing up on the Delphi, where can detect automatically that a new sample has been inserted).